src/containers/ReceiveWalletSelectionContainer.jsx

```diff
@@ -109,10 +130,12 @@ class ReceiveWalletSelectionContainer extends Component {
 const checkWalletConnectionSelector = createLoadingSelector([
   'CHECK_CLOUD_WALLET_CONNECTION',
   'CHECK_METAMASK_CONNECTION',
-  'CHECK_LEDGER_NANOS_CONNECTION'
+  'GET_LEDGER_WALLET_DATA'
```

Comment:
are we missing "CHECK_LEDGER_DEVICE_CONNECTION"

Reply:
No, CHECK_LEDGER_DEVICE_CONNECTION has it would create selector